src/utils/constants.js

```diff
@@ -55,6 +55,7 @@ export const CACHE_KEY_PRICE_DATA = 'aeternity-price-data'
 export const VISIBLE_KEYBLOCKS_LIMIT = 20
 export const VISIBLE_MICROBLOCKS_LIMIT = 30
 export const VISIBLE_TRANSACTIONS_LIMIT = 4
+export const NAVIGATION_HASH = '#navigation-open'
```

Comment:
wouldn't be better to keep the same naming as the variable in the code?
```suggestion
export const MENU_HASH = '#menu-open'
```